manager-parent/manager-v2/src/main/java/com/redhat/service/smartevents/manager/v2/api/ManagerConstraintViolationExceptionMapper.java

```diff
@@ -0,0 +1,35 @@
+package com.redhat.service.smartevents.manager.v2.api;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.validation.ConstraintViolation;
+import javax.ws.rs.ext.Provider;
+
+import com.redhat.service.smartevents.infra.core.exceptions.BridgeErrorService;
+import com.redhat.service.smartevents.infra.core.exceptions.mappers.ConstraintViolationExceptionMapper;
+import com.redhat.service.smartevents.infra.core.models.responses.ErrorResponse;
+import com.redhat.service.smartevents.infra.v2.api.V2APIConstants;
+
+@Provider
+@ApplicationScoped
+public class ManagerConstraintViolationExceptionMapper extends ConstraintViolationExceptionMapper {
```

Comment:
@r00ta Added... but probably affected by your refactoring.

It's presence here almost certainly guarantees I duplicate errors in the responses when running `v1` and `v2` together.